### DISCUSSION TITLE
[8 aprile 2024] Complessità

### DISCUSSION THREAD

**Discussion by Antaarees** [sapienzastudentsnetwork/automi-calcolabilita-complessita#27] [Complessità]
![photo_2024-04-10_17-16-39 (2)](https://github.com/sapienzastudentsnetwork/acc2324/assets/19594279/a5ef9634-44c6-48de-a407-4a3fc53f428c)

**Comment by Exyss** [comment]
Per il primo punto, ripropongo la soluzione presente sui miei appunti:

![image](https://github.com/user-attachments/assets/62166c6b-46a3-431d-a795-40d35774af11)

----

La dimostrazione richiesta nel secondo punto è presente sul libro di testo
